doc/paper/markdown/outline.tex.md

```diff
@@ -0,0 +1,1293 @@
+# Introduction
+
+is a decentralized permissionless smart contract platform biased towards
+low-latency management of assets. It uses the Move programming language
+to define assets as objects that may be owned by an address. Move
+programs define operations on these typed objects including custom rules
+for their creation, the transfer of these assets to new owners, and
+operations that mutate assets.
+
+is maintained by a permissionless set of authorities that play a role
```

Comment:
Same. I suspect the LaTex paper uses a macro `\sysname` referring to 'Suis' that has not been converted correctly.

Reply:
Yes, this comes from the intro file:

```
\sysname is a decentralized permissionless smart contract platform biased towards low-latency management of assets.
%
```

Reply:
See the issue for (not-much) progress on this front:
https://github.com/MystenLabs/sui/issues/1368